src/main/java/firebase/sdk/dashboard/data/User.java

```diff
@@ -0,0 +1,32 @@
+package firebase.sdk.dashboard.data;
```

Comment:
I hope this User class is outdated & #18  has the latest version. Hence I am skipping the review of User class in this PR.